### PR TITLE
[WIP] step-registry: add workflows for all test suites

### PR DESCRIPTION
**Pull request by bbguimaraes**

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+workflow:
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml:1]: Do we want to invent a new syntax for "copy this previous workflow but override" by re-using the logic we have for test cases in a singular `ci-operator` configuration?

Comment (bbguimaraes) [reply ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml:1]: I think the complexity of the implementation (execution, validation, resolution, rehearsals, …) does not justify it for this case. The steps are already factored into chains, so there is only three lines of repeated content in a few files that are all in the same place and could be easily changed together.

If we found a more complex case that would give greater benefits, I'd be fine with implementing it then coming back and DRYing this.

**Comment by bbguimaraes** [comment]
Superseded by

- https://github.com/openshift/release/pull/9866
- https://github.com/openshift/release/pull/9867
- https://github.com/openshift/release/pull/9882
- https://github.com/openshift/release/pull/9914
- https://github.com/openshift/release/pull/9915

/close

**Pull request closed without merging**